docker/build/Dockerfile

```diff
@@ -1,12 +1,9 @@
-FROM alpine:3.10.5
+FROM golang:1.19.2-buster
 
 LABEL maintainer="Tom Manville<tom@kasten.io>"
 
-RUN apk add --update --no-cache ca-certificates bash git docker curl jq \
-    && update-ca-certificates \
-    && rm -rf /var/cache/apk/*
```

Comment:
is this change intended => omitting `rm -rf /var/cache/apk/*`

Reply:
@julio-lopez wait I did clean it up. Yes it was intended. Will there be an apk cache if we are using `apt-get`?

Reply:
Oh, I see. Not an apk cache, but a deb cache. So this needs to be updated for that, not removed altogether.